Evergreen/Apps/Get-ConEmu.ps1

```diff
@@ -0,0 +1,26 @@
+Function Get-ConEmu {
```

Comment:
Should this be named `Maximus5ConEmu`?

Reply:
Probably yes, I renamed it in [46dc44c](https://github.com/aaronparker/evergreen/pull/398/commits/46dc44c543ef5afd3bf19f0d10a53585af76804e)